installing/install_config/installation-types.adoc

```diff
@@ -6,6 +6,11 @@ toc::[]
 
 You can perform different types of installations on different platforms.
 
+[NOTE]
+====
+Not all installation options are currently available for all platforms.
```

Comment:
Maybe a dumb question here. Are the installation options available/not available the options in the tables below? 
_Not all installation options are currently available for all platforms, as shown in the following tables_

Reply:
As long as I keep the table up to date, yes. :D